<ui version="4.0" >
 <class>App</class>
 <widget class="QDialog" name="App" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>[[[windowTitle]]]</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../src/autorun.qrc" >:/resource1/autorun.ico</iconset>
  </property>
  <property name="layoutDirection" >
   <enum>[[[layoutDirection]]]</enum>
  </property>
  <widget class="QLabel" name="label_logo" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap resource="../src/autorun.qrc" >:/resource1/board.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_run" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>210</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
   <property name="layoutDirection" >
    <enum>[[[layoutDirection]]]</enum>
   </property>
   <property name="text" >
    <string>[[[run]]]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_install" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>240</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
   <property name="layoutDirection" >
    <enum>[[[layoutDirection]]]</enum>
   </property>
   <property name="text" >
    <string>[[[install]]]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_quit" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>270</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
   <property name="layoutDirection" >
    <enum>[[[layoutDirection]]]</enum>
   </property>
   <property name="text" >
    <string>[[[quit]]]</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../src/autorun.qrc" />
 </resources>
 <connections/>
</ui>
